components/user/UserTabs.js

```diff
@@ -49,7 +49,7 @@ export default function UserTabs({ tabs, setTabs }) {
                 )}
                 aria-current={tab.current ? "page" : undefined}
               >
-                {tab.name} ({tab.total})
+                {tab.name} {tab.total && `(${tab.total})`}
```

Comment:
Added this change because it doesn't make sense to display this for the Eddiehub Stats